.github/workflows/release.yml

```diff
@@ -6,15 +6,15 @@ on:
   workflow_dispatch:
 jobs:
   release:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
Is there a reason you updated the Ubuntu version? Then it wouldn't run on Ubuntu 18.

Reply:
there is no reason for that just because my server was v20 i change it to make sure everything is ok, and I think most of the users have 20+ , you can change it if you want